ui/static/js/fortio_chart.js

```diff
@@ -446,9 +446,13 @@ function getCurrentChart () {
   return currentChart
 }
 
+var timeoutID = 0
 function updateChart (chart = getCurrentChart()) {
-  updateQueryString()
   updateChartOptions(chart)
+  if (timeoutID > 0) {
+    clearTimeout(timeoutID)
+  }
+  timeoutID = setTimeout("updateQueryString()", 750)
```

Comment:
I think it'd be more kosher to hook into the afterAnimation function that we found but seems good enough for the state of things.

Reply:
except I tried and it didn't work (because I think it's a different animation : https://github.com/istio/fortio/pull/243#issuecomment-385668354